src/neuroglancer/chunk_manager/backend.ts

```diff
@@ -861,3 +986,8 @@ registerRPC(CHUNK_SOURCE_INVALIDATE_RPC_ID, function(x) {
   const source = <ChunkSource>this.get(x['id']);
   source.chunkManager.queueManager.invalidateSourceCache(source);
 });
+
```

Comment:
I think this is no longer needed, but you could re-purpose CHUNK_SOURCE_FETCH_RPC_ID for the promise rpcs.

Reply:
Nevermind, I thought you were still using promiseInvoke

Reply:
Ah, yeah. I have an incoming update to this PR...